python_ta/contracts/__init__.py

```diff
@@ -176,6 +180,15 @@ def _check_function_contracts(wrapped, instance, args, kwargs):
     return r
 
 
+def _get_argument_suggestions(arg: Any, annotation: type) -> str:
+    """Returns potential suggestions for the given arg and its annotation"""
+    suggestions = ""
+    if isinstance(arg, type) and annotation in arg.mro():
```

Comment:
You can use the `issubclass` function (which also returns `True` if its two arguments are the same).